roles/network_plugin/contiv/defaults/main.yml

```diff
@@ -15,6 +15,9 @@ contiv_etcd_endpoints: |-
     contiv_etcd{{ loop.index }}=http://{{ hostvars[host]['ip'] | default(hostvars[host].ansible_default_ipv4['address']) }}:{{ contiv_etcd_peer_port }}{% if not loop.last %},{% endif %}
   {%- endfor %}
 
+contiv_etcd_init_image_repo: ferest/etcd-initer
```

Comment:
I think this should be removed. Or you suddenly have to maintain it multiple places.

Reply:
Arff you're right, thx. I guess I'm not with it these days... Fixed in 11d87ecc